monsieurbiz/sylius-search-plugin/1.0/config/packages/monsieurbiz_sylius_search_plugin.yaml

```diff
@@ -0,0 +1,24 @@
+imports:
+    - { resource: "@MonsieurBizSyliusSearchPlugin/Resources/config/config.yaml" }
+
+monsieur_biz_sylius_search:
+    files:
+        search: '%kernel.project_dir%/vendor/monsieurbiz/sylius-search-plugin/src/Resources/config/elasticsearch/queries/search.yaml'
+        instant: '%kernel.project_dir%/vendor/monsieurbiz/sylius-search-plugin/src/Resources/config/elasticsearch/queries/instant.yaml'
+        taxon: '%kernel.project_dir%/vendor/monsieurbiz/sylius-search-plugin/src/Resources/config/elasticsearch/queries/taxon.yaml'
+    documentable_classes :
+        - 'App\Entity\Product\Product'
+    grid:
+        limits:
+            taxon: [9, 18, 27]
+            search: [9, 18, 27]
+        default_limit:
+            taxon: 9
+            search: 9
+            instant: 10
+        sorting:
+            taxon: ['name', 'price', 'created_at']
+            search: ['name', 'price', 'created_at']
+        filters:
+            apply_manually: false # Will refresh the filters depending on applied filters after you apply it manually
+            use_main_taxon: true # Use main taxon for the taxon filter, else use the taxons
```

Comment:
Are these all default values?

Reply:
Yes

Reply:
@maximehuran how about you put that configuration in the source code and just import it here?

Reply:
I wanna check because the v1 of this plugin is not updated since we released the v2